simplemap.php

```diff
@@ -46,16 +46,10 @@
 define( 'SIMPLEMAP_URL', plugins_url() . '/' . $simplemap_dir );
 
 // Map HOST
-if ( ! defined( 'SIMPLEMAP_MAPS_JS_API' ) ) {
-	// TODO: Evaluate removing scheme check as scheme-less urls can be enqueued in 3.3 (WP #16560)
-	$scheme = 'http:';
-	if ( ! empty( $_SERVER['HTTPS'] ) && $_SERVER['HTTPS'] !== 'off' ) {
-		$scheme = 'https:';
-	}
+$scheme = 'https:';//Google APIs no longer work via HTTP
```

Comment:
I believe the http or https does need to match with both web servers. I may be wrong and/or something changed, but I believe it may have been more so with (website) https to http.

Reply:
@EkoJR I checked and that constant is only used to construct the request URL for geocode_location() which executes the API request so we're good here.

Reply:
Ok, just something I remember in the past. Can't mix http and https with REST/API calls.